src/components/InfoCard.tsx

```diff
@@ -61,29 +70,46 @@ const StyledInfoCard = styled.div`
   }
 
   @media (min-width: 768px) {
+    margin-bottom: 0rem;
+
     & > div > img {
       border-radius: 0rem;
     }
   }
+`;
 
-`
-
-const ButtonStyled = styled.button`
+export const StyledButton = styled.button`
```

Comment:
Unify your components. You created a button component and now you have another here or are they different?

Reply:
It has been removed.